cupy/_io/formatting.py

```diff
@@ -63,3 +63,14 @@ def format_float_positional(x, *args, **kwargs):
 
     """
     return numpy.format_float_positional(cupy.asnumpy(x), *args, **kwargs)
+
+
+def format_float_scientific(x, *args, **kwargs):
```

Comment:
Would you make it explicitly have the same parameters as those of `numpy.format_float_scientific`? I guess you followed the current implementation of `cupy.format_float_positional`, but I think it also should be fixed to have explicit parameters as its numpy counterpart.

Reply:
That's true in general, but in this case, I think it's ok to keep as-is.

When I reviewed `format_float_positional` in #6308 I asked to use `*args, **kwargs` because several options are only available in recent NumPy versions (e.g., `min_digits` added in NumPy 1.21.0), and we don't want to pay the cost to fill the gap between NumPy versions by ourself considering the importance of these "print" features.

Reply:
Makes sense. Then let me kick the CI to merge this soon!